content/english/csharp-guess-the-word/_index.md

```diff
@@ -16,27 +16,27 @@ The goal of today's activity is to learn C# skills and incorporate your new-foun
 
 The goal of today's activity is to learn C# skills and incorporate your new-found knowledge to create a Guess The Word game! Let's get the game set up.
 
-Today you will be using Replit. Before starting, you will need to sign up for a Replit account, if you do not yet have one. 
+Today you will be using .NET Fiddle. Before starting, you will need to sign up for a .NET Fiddle account, if you do not yet have one. 
 
-Click the button below to sign up for a Replit account - you will need an email account in order to sign up. If you already have a Replit account, you can login to your account using the below button as well.
```

Comment:
Let's mark the sign up process as optional by surrounding with a {{ % notice tip % }} and {{ % /notice % }}